highlevel/simulation/entity/event.py

```diff
@@ -10,6 +10,7 @@ class EventType(Enum):
     """
     Event type.
     """
+    WHEEL_SPEED = 'WHEEL_SPEED'
     MOVE_WHEEL = 'MOVE_WHEEL'
     MOVEMENT_DONE = 'MOVEMENT_DONE'
```

Comment:
movement done and move_wheels are gone, no?

remove?

Reply:
not really related, we will have to see later if the event queue is even needed anymore.

Reply:
Mh... maybe we can remove the queue then, since this is tied to the arch refactoring that you are doing...